.github/workflows/release-doc.yml

```diff
@@ -17,6 +17,6 @@ jobs:
         env:
           release_tag: ${{ github.ref }}
       - name: Create Release
-        uses: softprops/action-gh-release@v0.1.14
+        uses: softprops/action-gh-release@v1
```

Comment:
@atymchuk We need to add new version to allow list

Reply:
> @atymchuk We need to add new version to allow list

Done - @v1 is available.

Reply:
Thanks!